autoscheduler/frontend/src/tests/redux/CourseCards.test.ts

```diff
@@ -274,4 +284,109 @@ describe('Course Cards Redux', () => {
       });
     });
   });
+
+  describe('deletes all course cards and resets numCardsCreated', () => {
+    test('when clearCourseCards is dispatched', () => {
```

Comment:
```suggestion
  describe('clearCourseCards', () => {
    test('deletes all course cards and resets numCardsCreated, () => {
```

So while this does sort of invert our typical test structure, I think I prefer having like the "category" as the out-most `describe` as it groups all the test for that action under the same block (i.e. it specifies the system under test more directly). Also similar to how the first word/phase after `test_` in our Python tests are the system under test. Same thing with the rests of the tests